docs/hardhat-network/README.md

```diff
@@ -82,164 +85,37 @@ When a transaction fails without a reason, Hardhat Network will create a clear e
 
 - Trying to deploy a contract that exceeds the bytecode size limit imposed by [EIP-170](https://eips.ethereum.org/EIPS/eip-170)
```

Comment:
I wonder if we should drop this list, or reduce it to only three or four items that showcase part of the functionality (and maybe move the full list to a more referencish section, assuming this is something useful to have, although my guess is that it doesn't add a lot of value).

Reply:
I kinda like it here.  I think you're right that it doesn't add enough value to be added to the reference or anywhere else, but it *does* add value here in that it shows how many situations are covered, helping to drive home the answer to "Why would I want to use it?"  While it may feel nicely succinct to only show a few examples, I think it's easy enough for people to just scroll through it as it is right now.